custom-vu/src/main/kotlin/jces1209/vu/action/ProjectIssueNavigatorAction.kt

```diff
@@ -0,0 +1,34 @@
+package jces1209.vu.action
+
+import com.atlassian.performance.tools.jiraactions.api.BROWSE_PROJECTS
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import com.atlassian.performance.tools.jiraactions.api.action.Action
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import com.atlassian.performance.tools.jiraactions.api.memories.ProjectMemory
+import jces1209.vu.page.project.ProjectIssueNavigatorPage
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+
+class ProjectIssueNavigatorAction(
```

Comment:
Other actions are named like commands
```suggestion
class BrowseProjectIssues(
```